src/viewport/viewport.model.ts

```diff
@@ -37,6 +28,19 @@ export enum DeviceType {
 export interface UxViewportOptions {
 	/** Polling speed on resizing (in milliseconds). e.g. the higher the number the longer it takes to recalculate. */
 	resizePollingSpeed: number;
+	/** A dictionary of custom breakpoints where the value is the width threshold. */
```

Comment:
include that as i find it quite useful myself 😅 
```suggestion
	/** A dictionary of custom breakpoints where the value is the width threshold. e.g. given width '1000' and `medium` is set to '992' => `large` */
```